### ISSUE TITLE
Can't install the program

### ISSUE THREAD

**Issue by patelmmp** [tdhoward/COMpipe#1]
Hi,

I am trying to install the program on Win 10 Pro. When I run .exe as an administrator, nothing happens. It doesn't install.

Thanks.

**Comment by tdhoward** [comment]
This is not an installer, just a command line utility. If you run it from the command line it will give information about the configuration options.

**Comment by sparkyhotm** [comment]
Wow - it was a LONG journey to find something free that does this.  Thanks for that.  I just tested on my Hyper-V host 2012 R2 talking to my 2012 R2 Client and it worked fine.  Things I learned: Run as admin.  Don't configure the CLIENT VM COM PORT with a name COMx - COMx is a reserved system device name and makes things not work.  Name your pipe PORTx.  Now, I have to find something that will do this, and run as a service.  Dealing with a serial port in Hyper-V is a PITA.  
Again - Thank you for making this available.

**Comment by sparkyhotm** [comment]
Ok...further testing revels this is limited to 9600 Baud.  Is there a way to pass a port configuration?  I really need it to go up to 115200, 8, 1, N.

**Comment by tdhoward** [comment]
Sure, I'll work on that.  Give me a few hours... :-)

**Comment by sparkyhotm** [comment]
Thanks - I figured out how to re-compile it myself.  More testing revels that the program/process is somehow corrupting/loosing data even at 2400 baud.  I'm using this on a 2012 R2 Datacenter Hyper-V host.  Normal typing goes thru fine, but hyperterm on both ends using  ZModem transfers over the path fail (even at 2400 Baud).  My guess is buffer overruns, but I would be guessing.  I've looked at the code some, but I'm not a "current" programmer, so I have to try other solutions like USB redirection, or Serial-over-ethernet.  I have high hopes for your later refinements.  Thanks for your efforts to fill a hole Microsoft left in Hyper-V.  This is the closest thing I've found to a simple solution.

BTW - I'd recommend allowing the passing of the port settings on the command line, like -c \\.\com1,2400,8,1,N,XONXOFF -p \\.\\pipe\port1 - something like that.
